cluster/gce/gci/configure-helper.sh

```diff
@@ -2126,7 +2127,8 @@ function start-kube-scheduler {
     create-kube-scheduler-config
```

Comment:
what happens in this branch?

Reply:
I am not sure, looks like the file `KUBE_SCHEDULER_CONFIG` is managed by some other users. In addition, `${KUBE_SCHEDULER_CONFIG}` is not set currently: https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/96550/pull-kubernetes-e2e-gce-100-performance/1327123753032749056/artifacts/e2e-96550-95a39-master/kube-scheduler.log (`--config=""`)

Reply:
not in CI, but by some launchers, it can be

@ahg-g, how does kube-scheduler configure these two flags when running from a config file?

Reply:
nothing different: https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-scheduler/app/options/options.go#L216

Reply:
shouldn't we pass those flags in both cases, when component-config is passed and when not?

Reply:
given that the content of `KUBE_SCHEDULER_CONFIG` is managed by other users and I guess it is not desired to append these two flags in this script, I think it is okay to left as it is and let the user set these flags in the `KUBE_SCHEDULER_CONFIG`.

Reply:
> and let the user set these flags in the `KUBE_SCHEDULER_CONFIG`.

can those flags be set via that config file?

Reply:
> can those flags be set via that config file?

hmmm after looking into the source code, I don't think so..:
https://github.com/kubernetes/kubernetes/blob/0913f8a54cc68164d5182796476a9f608321ac17/pkg/scheduler/apis/config/types.go#L55-L112
I should take a look first..

And regarding the config file, user might decide to set the path to kubeconfig by setting `.clientConnection.kubeconfig` in the config file, and it is not quite easy to extract that path from `KUBE_SCHEDULER_CONFIG`.

Reply:
As now `--kubeconfig` could be set via the config file, is it reasonable to set `--authentication-kubeconfig` and `--authorization-kubeconfig` via the config file?
cc @ahg-g

Reply:
Adding those flags to the config is probably a question to component config working group, those seem to be generic enough to be placed in a common type embedded in all configs.

Reply:
@liggitt now that the v1.20 restriction has been lifted, I want to pick up the work now, is it required to add those flags to kube-scheduler config?

Reply:
It has to work whether we start from a config file or from flags. I'll defer to the scheduler team on how to accomplish that.

Reply:
@ahg-g @liggitt Now this should work for both cases, PTAL